components/TabBar/TabBar.css

```diff
@@ -11,6 +11,6 @@
 
 .tab {
   font-weight: 600 !important;
-  font-size: 11 !important;
+  font-size: 12px !important;
   color: var(--primaryTextColor) !important;
 }
```

Comment:
Remember we only need !important for when we are overriding element inline styles (such as those applied to things by material-ui). This may be one such case, I didn't check, but we should try to avoid when possible.